### ISSUE TITLE
Question: Are co-installations for "enabled" addons or all addons installed?

### ISSUE THREAD

**Issue by birdsarah** [mozilla/taar-lite#50]
We should add that definition to the docs.

**Comment by birdsarah** [comment]
ping @crankycoder @mlopatka

**Comment by mlopatka** [comment]
enabled only
https://github.com/mozilla/python_mozetl/blob/4bd8ceed909ae779826c383cbf7659b5dbf5869b/mozetl/taar/taar_lite_guidguid.py#L62

**Comment by dzeber** [comment]
Actually, we only record Telemetry for enabled add-ons. This was a decision made back when we developed Unified Telemetry in 2016.

In other words, if you have add-ons that are installed but disabled, they are not included in your Telemetry ping. You can see this by looking at your own `about:telemetry`. The fields `app_disabled` and `user_disabled` are included in Telemetry because they are part of the standard add-on fields that get pulled from the Add-ons Manager, but if you look at our collected data they are always `False`.

**Comment by mlopatka** [comment]
@dzeber well... TIL.
Actually, wow this implies that from a (Longitudinal) Telemetry standpoint disabling is equivalent to an uninstall.

**Comment by dzeber** [comment]
@mlopatka That's right, and this occurs in the raw pings, meaning that all of our data infrastructure built on top uses the same assumption. If you're interested, we can chat offline about the background for this decision and ideas that have been floated to capture disabling.
